interface/src/common/statusBar/statusBar.js

```diff
@@ -0,0 +1,12 @@
+angular.module('statusBar', [])
```

Comment:
As usual, StackOverflow provides a good explanation. Apparently the hyphenated-names are sometimes called kebab-case. LOL!

http://stackoverflow.com/questions/33460274/angularjs-directive-naming-conventions

Where I probably saw this was in the Angular tutorial here: https://docs.angularjs.org/tutorial/step_00#what-is-the-code-doing-